src/components/job-card.jsx

```
@@ -0,0 +1,66 @@
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "./ui/card";
+import { Link } from "react-router-dom";
+import useFetch from "@/hooks/use-fetch";
+import { deleteJob } from "@/api/apiJobs";
+import { useUser } from "@clerk/clerk-react";
+import { BarLoader } from "react-spinners";
+
+const JobCard = ({ job, onJobAction = () => { }, isMyJob = false }) => {
+  const { user } = useUser();
```

ESLint

[error] 'user' is assigned a value but never used.

(no-unused-vars)